cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -200,4 +206,10 @@ func init() {
 		log.Fatal(err)
 	}
 
+	Cmd.Flags().Int("opReturn", 0, "Text which will be added to an OP_RETURN output. If empty, no OP_RETURN output will be added")
```

Comment:
This flag needs to have type string
```suggestion
	Cmd.Flags().String("opReturn", "", "Text which will be added to an OP_RETURN output. If empty, no OP_RETURN output will be added")
```

Reply:
Done